### ISSUE TITLE
Mesh setup: "scanning for mesh network" screen should require action

### ISSUE THREAD

**Issue by ScruffR** [particle-iot/photon-tinker-android#52]
I stumbled over this myself several times and have seen users thrown off by it too.
When adding a device to a mesh network you get the "scanning for mesh network" screen which eventually shows the found networks but there is no prompt to actually select one of the found networks and the spinner keeps spinning one might consider the app stuck.

As soon a network has been found a prompt to actually click on the network name may help there.

**Comment by jensck** [comment]
Did you have any kind of prompt in mind?  I wonder if just altering/adding some header text would be enough.

As for the spinner, it stays there to indicate that scanning is always ongoing, since the state of available mesh networks is transitory.

**Comment by ScruffR** [comment]
Altering the header text should suffice.
e.g. _"Please select mesh network to join / (scanning for available mesh)"_

The focus of that step is not the scanning in itself (which would not require the user to do anything but _wait_), but should inform the user that _his_ action is required.

**Comment by andyw-lala** [comment]
I agree with @ScruffR that the current behaviour is misleading/sub-optimal.

Providing a hint that the user can select as soon as they see the network the want would be good, especially since the spinner is active (usually means the user should wait.)
